ca_tools/scripts/control_node.py

```diff
@@ -44,142 +39,133 @@ class ctrl_Node:
+            self._my_pub.publish(aux)
 
-    def stop_turning(self):
-        self.stop()
-        self.moveForward()
-        self.state=FORWARD
+    def _stop_Turning(self):
+        self._stop()
+        self._moveForward()
+        self._state=FORWARD
 
-    def stop(self):
+    def _stop(self):
             aux = Twist()
-            aux.angular.x=0.0
-            aux.angular.y=0.0
-            aux.angular.z=0.0
-            aux.linear.x=0.0
-            aux.linear.y=0.0
-            aux.linear.z=0.0
-            self.my_pub.publish(aux)
+            self._my_pub.publish(aux)
     
-    def moveForward(self):
+    def _moveForward(self):
         aux = Twist()
-        aux.angular.x=0.0
-        aux.angular.y=0.0
-        aux.angular.z=0.0
         aux.linear.x=0.2
-        aux.linear.y=0.0
-        aux.linear.z=0.0
-        self.my_pub.publish(aux)
-
-    def forward(self):
-        if (self.reachedPosition()):
-            self.stop()
-            self.state=STOP_FORWARD
-            self.goal_num+=1
-            self.goal_num%=4
+        self._my_pub.publish(aux)
+
+    def _forward(self):
+        if (self._reachedPosition()):
+            self._stop()
+            self._state=STOP_FORWARD
+            self._goal_num+=1
+            self._goal_num%=4
         else:
-            self.setGoalAngle()
-            if (not self.reachedAngle()):
-                self.state=STOP_FORWARD
-                self.stop()
+            self._setGoalAngle()
+            if (not self._reachedAngle()):
+                self._state=STOP_FORWARD
+                self._stop()
             else:
-                self.moveForward()
-    
-    def stop_forward(self):
-         self.stop()
-         self.setGoalAngle()
-         self.state=TURNING
-
-    def setGoalAngle(self):
-        self.my_angleGoal=math.atan2((self.my_goals[self.goal_num].y-self.my_pose.y),self.my_goals[self.goal_num].x-self.my_pose.x)
-        rospy.loginfo("Difference between angles is %f,",(self.my_angleGoal-self.my_pose.theta))
+                self._moveForward()
+
+    def _diffAngle(self):
+        return (self._my_angleGoal-self._my_pose.theta)
+
+    def _angleIsBig(self):
+        return ((self._my_angleGoal-self._my_pose.theta)>1) and ((self._my_angleGoal-self._my_pose.theta)<5.28)
+
+
+
+    def _stop_Forward(self):
+         self._stop()
+         self._setGoalAngle()
+         self._state=TURNING
+
+    def _setGoalAngle(self):
+        self._my_angleGoal=math.atan2((self._my_goals[self._goal_num].y-self._my_pose.y),self._my_goals[self._goal_num].x-self._my_pose.x)
+        rospy.loginfo("Difference between angles is %f,",(self._my_angleGoal-self._my_pose.theta))
     
-    def reachedAngle(self):
-        return ((abs(self.my_angleGoal-self.my_pose.theta))<0.2 or (abs(self.my_angleGoal-self.my_pose.theta))>6.04)
+    def _reachedAngle(self):
+        return ((abs(self._my_angleGoal-self._my_pose.theta))<0.2 or (abs(self._my_angleGoal-self._my_pose.theta))>6.04)
```

Comment:
There are some constant numbers that I recommend you to move them before the constructor.

Reply:
Adressed